### DISCUSSION TITLE
Is there information on deprecated or obsolete elements and properties in MDN?

### DISCUSSION THREAD

**Discussion by debiru** [mdn/mdn-community#344] [Content] [answered]
Sometimes you are researching HTML or CSS and want information about something that existed in the past but is no longer in use today.

For example, let's say you want to know about the specification of the `text-wrap` property. In fact, however, no description of the `text-wrap` property could be found in MDN.

What policy determines whether information is available on MDN for each HTML and CSS element and property? `text-wrap` is an example, but I would like to know how you decide whether to publish it or not.

**Comment by teoli2003** [comment]
We document features implemented in at least one of the three major engines (Chromium, WebKit, Gecko) over the last two years, regardless of their spec status.

A feature that has not yet been implemented in a released browser or has been removed from all three engines for more than two years is not documented on MDN (There are exceptions, but they are errors)

**Reply by debiru** [reply]
@teoli2003 

It has been some time since the topic was created, but I have a question.

> ... over the last two years,

> ... for more than two years is not documented on MDN

Is this policy of deleting after "two years" clearly stated anywhere? It does not appear to be written in the page of [Experimental, deprecated, and obsolete](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete).

However, I actually found [a pull request](https://github.com/mdn/content/pull/26850) stating that it will be removed after 2 years. I am wondering where this policy originated.

**Comment by dipikabh** [comment, marked as answer]
What @teoli2003 said ☝️. Also our policy around this topic is documented here: [Experimental, deprecated, and obsolete](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete )

**Reply by debiru** [reply]
Thank you for letting us know that the document existed on MDN. I found it helpful.
